docs/integrations/sources/s3.md

```diff
@@ -20,12 +19,12 @@ This page contains the setup guide and reference information for the Amazon S3 s
 
 1. [Log into your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account.
 2. In the left navigation bar, click **<Sources/Destinations>**. In the top-right corner, click **+new source/destination**.
-3. On the Set up the <source/destination> page, enter the name for the <connector name> connector and select **<connector name>** from the <Source/Destination> type dropdown.
```

Comment:
this was breaking docusaurus

Reply:
I think the JSX translation layer is afraid of anything that vaguely resembles HTML tags